compiler+runtime/src/jank/clojure/core.jank

```diff
@@ -1918,20 +1918,32 @@
    f must be free of side-effects"
   clojure.core-native/iterate)
 
-; TODO: integer_range
-(def range
+(defn range
   "Returns a lazy seq of nums from start (inclusive) to end
    (exclusive), by step, where start defaults to 0, step to 1, and end to
    infinity. When step is equal to 0, returns an infinite sequence of
    start. When start is equal to end, returns empty list."
-  clojure.core-native/range)
-
-(def nth
-  "Returns the value at the index. get returns nil if index out of
+  ([]
+   (iterate inc 0))
+  ([end]
+   (if (integer? end)
+     (clojure.core-native/integer_range end)
+     (clojure.core-native/range end)))
+  ([start end]
+   (if (and (integer? start) (integer? end))
+     (clojure.core-native/integer_range start end)
+     (clojure.core-native/range start end)))
+  ([start end step]
+   (if (and (integer? start) (integer? end) (integer? step))
+     (clojure.core-native/integer_range start end step)
+     (clojure.core-native/range start end step))))
+
+  (def nth
+   "Returns the value at the index. get returns nil if index out of
    bounds, nth throws an exception unless not-found is supplied.  nth
    also works for strings, Java arrays, regex Matchers and Lists, and,
    in O(n) time, for sequences."
-  clojure.core-native/nth)
+   clojure.core-native/nth)
```

Comment:
Looks to me like `(def nth` was indented here, for some reason.

Reply:
Formatting issue, fixed it!